ktouch-lesson-generator.py

```diff
@@ -14,7 +14,10 @@
   -p, --plain-text                           Output the lessons in plain text instead of XML
   -w NUM, --word-wrap NUM                    Wrap lesson text at this length. [default: 60]
   -l NUM, --characters-per-lesson NUM        Number of characters in a lesson. [default: 2000]
+      --course-description STR               Sets the description for the course [default: ""]
```

Comment:
For these three new parameters you cannot set the default here because that string e.g. `""` is used as default value and you get this as result:
```
<title>"KTouch-Generator-{shortid}"</title>
<description>""</description>
<keyboardLayout>""</keyboardLayout>
```
You can write the default as text not withing square brackets and then process the default value that is `None`.